package.json

```diff
@@ -29,13 +29,14 @@
     "strip-json-comments": "1.0.2"
   },
   "devDependencies": {
-    "mocha": "*",
-    "expect.js": "*"
+    "concat-stream": "^1.5.0",
+    "expect.js": "*",
+    "mocha": "*"
   },
   "bin": {
     "jsinspect": "./bin/jsinspect"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha -R spec spec spec/reporters"
+    "test": "./node_modules/.bin/mocha -R spec spec/reporters"
```

Comment:
Can you undo this? This change makes it so that none of other unit tests (nodeUtils, Match, Inspector) run